msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Preapproval of RSC permissions
+description: In this article, learn more about to create, manage, and delete preapprovals of RSC enabled app in Microsoft Teams.
+ms.localizationpriority: medium
+author: MSFTRickyCastaneda
+ms.author: rcastaneda
+ms.topic: conceptual
+ms.date: 08/29/2023
+---
+
+# Preapproval of RSC permissions
+
+Resource-Specific Consent (RSC) permission preapproval provides admins with detailed control over the RSC permissions that an app can request during installation. The app only receives RSC permissions at the installation time. However, using preapproval policies, admins can predetermine the maximum permissions that an app can request from the end user and secure consent during installation.
+
+To successfully create and use a preapproval policy for an app, admins must ensure the app can be installed in their organization. For more information, see [managing apps in Teams Admin Center](/microsoftteams/apps-in-teams). Preapproval policies don't affect apps blocked through Teams Admin Center.
```

Comment:
For an RSC permission pre-approval policy to take effect for an app, the admin must first ensure that the app is enabled (installable) in their organization. If the app is not installable, then the permission pre-approval for that specific app becomes irrelevant. For more information on enabling apps in TAC, see [managing apps in Teams Admin Center](/microsoftteams/apps-in-teams).

Reply:
Updated as suggested